packages/fxa-content-server/app/styles/modules/_button-row.scss

```diff
@@ -73,6 +73,7 @@ button {
 
   &:disabled,
   &.disabled {
+    background: darken($button-background-color-default-hover, 5);
```

Comment:
I believe `$button-background-color-default` is the same color as this. Can we use that variable instead for consistency?

Reply:
@LZoog yeah, that makes more sense for consistency. Fixed that.